src/pages/journal/components/journal-navigation/__tests__/journal-navigation.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { JournalNavigationComponent } from '../journal-navigation';
+import { IonicModule, Config } from 'ionic-angular';
+import { ConfigMock } from 'ionic-mocks-jest';
+import { StoreModule } from '@ngrx/store';
+import { journalReducer } from '../../../journal.reducer';
+
+describe('JournalNavigationComponent', () => {
+  let component: JournalNavigationComponent;
+  let fixture: ComponentFixture<JournalNavigationComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ JournalNavigationComponent ],
+      imports: [
+        IonicModule,
+        StoreModule.forRoot({
+          journal: journalReducer
+        }),
+      ],
+      providers: [
+        { provide: Config, useFactory: () => ConfigMock.instance() }
+      ]
+    }).compileComponents().then(() => {
+      fixture = TestBed.createComponent(JournalNavigationComponent);
+      component = fixture.componentInstance;
+    });
+  }));
+
+  describe('Class', () => {
+    it('should create', () => {
+      expect(component).toBeDefined();
+    });
+  });
+
+  describe('DOM', () => {
+    
```

Comment:
Should we not have some test here based on the 3 ngIf's in the html

Reply:
Good that you mentioned, writing some tests for those conditions.

Reply:
Implementing tests after the S&T and will bring them in with a different PR due to time constraints.